software/oort-config/common/_MANIFEST.csv

```diff
@@ -10,11 +10,20 @@ root,wheel,700,/root
 
 # OVERRIDES START
 root,wheel,700,/_AUTOGEN_install.site
+root,wheel,644,/etc/dhcpd.conf
 root,wheel,644,/etc/doas.conf
+root,wheel,640,/etc/hostname.re0.conf
+root,wheel,640,/etc/hostname.re1.conf
+root,wheel,640,/etc/hostname.wlan0.conf
+root,wheel,600,/etc/pf.conf
+root,wheel,640,/etc/rc.conf.local
 root,wheel,640,/etc/rc.firsttime
+root,wheel,644,/etc/resolv.conf
+root,wheel,644,/etc/sysctl.conf
 ohsnap,ohsnap,600,/home/ohsnap/_DOT.zprofile
 ohsnap,ohsnap,600,/home/ohsnap/_DOT.zshrc
 root,wheel,700,/root/post_install.sh
 root,wheel,700,/root/post_install_packages.sh
 root,wheel,600,/root/_AUTOGEN_post_install_hosts_append
-root,wheel,600,/root/_AUTOGEN_post_install_package_list.txt
\ No newline at end of file
+root,wheel,600,/root/_AUTOGEN_post_install_package_list.txt
+root,wheel,644,/var/unbound/etc/unbound.conf
```

Comment:
One quirk about oort in its current implementation is that it doesn’t automatically create intermediate directories between a parent directory and a leaf file. Each intermediate directory needs to be listed explicitly in the manifest. (Among other reasons, this requires giving explicit consideration to the permissions and ownership of each directory, so as to reduce chances of misconfiguration.) That’s the reason for the separate “# Preexisting directories” section at the top of the manifest. You’d want to add an entry for /var up there and then separate entries for /var/unbound and /var/unbound/etc down here before the /var/unbound/etc/unbound.conf line

Reply:
Ah, ok. I think it should be good now.